01-ArrayManipulation/03-CountDigits/countDegit.cpp

```diff
@@ -0,0 +1,36 @@
+#include<iostream>
+
+using namespace std;
+
+int tab[10] = {0,0,0,0,0,0,0,0,0,0};
+int n;
+
+int NbDigits(int n, int d) {
+  int count = 0;
+
+  while (n > 0) {
+
+    if (n % 10 == d)
+      count++;
+    n = n / 10;
+  }
+
+  return count;
+}
+
+int main() {
+  while (1) {
+    cin >> n;
+
+    for (int i = 0; i < 10; i++) {
+      tab[i] = tab[i] + NbDigits(n, i);
```

Comment:
`+=` is simpler, cleaner in this case.

Reply:
(in this form, it is easier to mistype one of the array indices by accident, and end up with something like `tab[i]=tab[j]+,,,`